pkgs/applications/misc/shoutrrr/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
```

Consider removing the `rec` for the `let in` synatx, see here https://nix.dev/guides/best-practices.html.
```suggestion
let
  version = "0.8.0";
in
buildGoModule  {
```

---

done